src/repository/opamRepositoryBackend.ml

```diff
@@ -126,7 +126,20 @@ let strip_repo_suffix patch =
   in
   {patch with operation}
 
-let get_diff parent_dir dir1 dir2 =
```

Comment:
`get_diff`is historically in `OpamRepositoryBackend`, but given the functions added, its use in another PR for pinning stuff, i think it should be moved. Maybe a `core/opamDiff.ml`?